homeassistant/components/kostal_plenticore/const.py

```diff
@@ -0,0 +1,545 @@
+"""Constants for the Kostal Plenticore Solar Inverter integration."""
+
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_ICON,
+    ATTR_UNIT_OF_MEASUREMENT,
+    DEVICE_CLASS_BATTERY,
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_POWER,
+    ENERGY_KILO_WATT_HOUR,
+    PERCENTAGE,
+    POWER_WATT,
+)
+
+DOMAIN = "kostal_plenticore"
+
+ATTR_VALUE = "value"
+ATTR_ENABLED_DEFAULT = "entity_registry_enabled_default"
+
+"""Defines all entities for process data.
```

Comment:
Please write comments with `# `. Multi-line strings are not used for code comments.